src/bowser.js

```diff
@@ -88,7 +88,8 @@
     }
     else if (iosdevice) {
       result = {
-        name : iosdevice == 'iphone' ? 'iPhone' : iosdevice == 'ipad' ? 'iPad' : 'iPod'
+        name : 'Safari'
```

Comment:
Browsers other than Safari could be used on iPhone / iPad; for instance, Chrome for iOS.